pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -69,8 +69,6 @@ var _ = Describe("Tuned", func() {
 			By("Populating hugepages cmdline")
 			Expect(cmdlineHugepages.MatchString(manifest)).To(BeTrue())
 			By("Populating empty additional kernel arguments cmdline")
-			Expect(manifest).To(ContainSubstring("cmdline_additionalArg="))
```

Comment:
in the original PR we added a test (which is the right thing to do), while in this backport we are making an existing test weaker (?) and not adding new tests. Can we improve somehow?

Reply:
its not weaker, but the cherry pick takes also this change:
`{{if .AdditionalArgs}}
cmdline_additionalArg=+{{.AdditionalArgs}} 
{{end}}`
Then it's not necessary anymore to check if the is empty additionalArgs in cmdline. In any case is better delete this change and continue as before. But, if:
`{{if .RealTime}}
cmdline_pstate=+intel_pstate=disable
{{end}}`

is added, this check:
Expect(manifest).To(ContainSubstring("cmdline_additionalArg="))``

Fails, instead, if:
`intel_pstate=disable`

is added to the existing line:
`{{if .RealTime}}
cmdline_realtime=+nohz_full=${isolated_cores} tsc=nowatchdog nosoftlockup nmi_watchdog=0 mce=off skew_tick=1 rcutree.kthread_prio=11 intel_pstate=disable
{{end}}`

this fails:
`Expect(cmdlineHugepages.MatchString(manifest)).To(BeTrue())`

About the added test: 4.12 has perPodPowerManagement and doesn't have REGEX to check tests. If I add a test this wouldn't be a cherry-pick. It will be almost a new PR, because there are many changes until here...
I can do it but not sure if it is the best solution.

Reply:
ok. Drifting too far from cherry-pick is indeed not good. If there's not a reasonable way to cherry-pick or backport tests, I guess we're good as we are.

Reply:
Thanks, I've tried to make the minimum possible changes but the test are different and changing things some of them get broke, also there is another hint in 4.12 that changes the cherry-pick.